aiomisc/dependency.py

```diff
@@ -0,0 +1,62 @@
+from collections import namedtuple
+from functools import wraps
+
+from aiodine.store import Store
+
+
+STORE = Store()
```

Comment:
Should you add thread local storage for this global? I can not imagine how do this works when entrypoint running in different threads.